tests/vhdlFile/test_token.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+import os
+import unittest
+
+import vsg.vhdlFile as vhdlFile
+from tests import utils
+
+lTokenFiles = os.scandir(os.path.dirname(__file__))
+
+lLrmUnits = [x.name for x in lTokenFiles if x.is_dir()]
+if "__pycache__" in lLrmUnits:
+    lLrmUnits.remove("__pycache__")
```

Comment:
Slight performance enhancement
```suggestion
with os.scandir(os.path.dirname(__file__)) as lTokenFiles:
    lLrmUnits = [entry.name for entry in lTokenFiles if entry.is_dir() and entry.name != "__pycache__"]
```

Reply:
Hi @larshb. Thanks for your suggestion. I've implemented it, but wrapped it up in a function as it's becoming quite difficult to read!

Reply:
Nice! Yeah, probably a good idea. Readability is more important anyway.